cli/src/test/java/com/devonfw/tools/ide/tool/Jmc/JmcTest.java

```diff
@@ -1,4 +1,4 @@
-package com.devonfw.tools.ide.Jmc;
+package com.devonfw.tools.ide.tool.Jmc;
```

Comment:
Please fix the package statement to fit to the path the file is located in (Eclipse would give you an error for this inconsistency, it seems that IntelliJ is very lax in this regard and does not complain). Also please rename `Jmc` package folder to `jmc` as Java-Package segments need to be `lowercase` and should avoid any `UPPERCASE` letters.
```suggestion
package com.devonfw.tools.ide.tool.jmc.Jmc;
```

Reply:
I was able to solve this on github with just my browser now.